src/tests/conformance/cmd/transaction.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{
+    protocol::{
+        codecs::binary::{BinaryMessage, Payload},
+        proto::TransactionStatus::TsCurrent,
+    },
+    setup::{constants::TESTNET_READY_TIMEOUT, testnet::TestNet},
+    tools::{
+        rpc::{submit_transaction, wait_for_account_data},
+        synth_node::SyntheticNode,
+    },
+};
+
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn c019_MT_TRANSACTION_node_should_broadcast_transaction_to_all_peers() {
+    // ZG-CONFORMANCE-019
+
+    const FIRST_NODE_ID: usize = 0;
+    const SECOND_NODE_ID: usize = 1;
```

Comment:
```suggestion
    const NODE_IDS: [usize; 2] = [0, 1];
```
and then use `NODE_IDS[0]` etc; it would be a bit more versatile